tests/migrations/test_operations.py

```diff
@@ -779,7 +779,7 @@ def test_rename_m2m_model_after_rename_field(self):
         Pony = project_state.apps.get_model(app_label, 'Pony')
         Jockey = project_state.apps.get_model(app_label, 'Jockey')
         PonyRider = project_state.apps.get_model(app_label, 'PonyRider')
-        # No "no such column" error means the column was renamed correctly.
+        # No "no such column" error means the column was not renamed correctly.
```

Comment:
Are you sure? The previous version looks correct to me, maybe:
```suggestion
        # "no such column" error means the column wasn't renamed correctly.
```

Reply:
🤦 you are right I missed the double negation.